kimenyikevin/FreeMentor
# #168362124, #168372117 set up database and Build sign up

#### What does this PR do?
- set up DataBase and Have sign up working with database
#### Description of Task to be completed?
Have the following EndPoints work with Database
- POST /api/v2/auth/signUp
#### How should this be manually tested?
- clone repo and cd into it, install the package first by run **npm i**  run **npm run test** in your terminal to see if all test are passing 
- Also, you can check above EndPoint in postman before you do so start a server in your terminal by run **npm run dev-start**
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#168362124, #168372117
#### Screenshots (if appropriate)
#### Questions:
